README.md

```diff
@@ -29,6 +29,9 @@ Array of words that will not be checked.
 "skipIfMatch": <<Array Of Strings>> default: []
 Array of Regular Expressions the plugin will try to match the js node element value (identifier, comment, string, string template, etc) and will not check the entire node content if matched, be careful in comments because if a part of the comment is matched the entire comment will not be checked, same for strings.
 i.e: "^[-\\w]+\/[-\\w\\.]+$" will ignore MIME types.
+
+"minLength": <<Number>> default: 1
+Words with a character-amount of less than the minLength will not be spell-checked.
```

Comment:
Please change to:
js node element value (identifier, comment, string, string template, etc) with a character-amount of less than the minLength will not be spell-checked.

Because we also divide the token for example for camelCalse or snakeCase and then check the spelling inside each word. And this option applies only to the complete node content.

Reply:
Looks like this PR does not exactly what I wanted to do.
I will do some changes so the code does what the description tells.

Reply:
Ok cool no problem, I think that you want to filter any subsring with less than x chars right? 
Not the entire node. So if a variable is idElement you don't want to spell check id. 
That is ok for me too.

Reply:
Yes that is the goal. PR will come soon.